src/settings.ts

```diff
@@ -580,6 +581,13 @@ export const SEARCH_DEBOUNCE_TIME = 200; // ms
 
 export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 12;
 
+export const USER_STATUSES = [
+  UserStatus.online,
+  UserStatus.busy,
+  UserStatus.offline,
+  UserStatus.custom,
```

Comment:
remove online/offline from this list, please as a person can't set it himself

Reply:
set status to online is the way to change status from `busy` to normal

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/9da797f41a65b4b551961457ff493326a6918da5